src/components/container/SurveyDataChart/SurveyDataChart.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useContext } from 'react'
+import { DateRangeContext } from '../../presentational/DateRangeCoordinator/DateRangeCoordinator'
+import { add, parseISO } from 'date-fns'
+import { SurveyAnswer } from '@careevolution/mydatahelps-js'
+import { WeekStartsOn, getMonthStart, getWeekStart } from '../../../helpers/get-interval-start'
+import TimeSeriesChart, { AreaChartOptions, BarChartOptions, LineChartOptions } from '../../presentational/TimeSeriesChart/TimeSeriesChart'
+import queryAllSurveyAnswers from '../../../helpers/query-all-survey-answers'
+import format from 'date-fns/format'
+import { useInitializeView } from '../../../helpers/Initialization'
+
+export interface SurveyAnswerChartParameters {
+    label: string;
+    surveyName?: string | string[];
+    stepIdentifier?: string | string[];
+    resultIdentifier: string | string[];
+}
+
+export interface SurveyDataChartProps {
+    title?: string
+    intervalType?: "Week" | "Month" | "SixMonth"
+    weekStartsOn?: WeekStartsOn
+    charts: SurveyAnswerChartParameters[],
+    chartType: "Line" | "Bar" | "Area"
+    options?: LineChartOptions | BarChartOptions | AreaChartOptions,
+    expectedDataInterval?: Duration,
+    previewDataProvider?: (startDate: Date, endDate: Date) => Promise<SurveyAnswer[][]>
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+function getDefaultIntervalStart(intervalType: "Week" | "Month" | "SixMonth", weekStartsOn?: WeekStartsOn) {
+    let intervalStart: Date;
+    if (intervalType === "Week") {
+        intervalStart = getWeekStart(weekStartsOn);
+    } else {
+        intervalStart = getMonthStart();
+    }
+    return intervalStart;
+}
+
+export default function SurveyDataChart(props:SurveyDataChartProps) {
+    let [currentData, setCurrentData] = useState<{ [key: string]: SurveyAnswer[] } | null>(null);
+    
+    const dateRangeContext = useContext<DateRangeContext | null>(DateRangeContext);
+    let intervalType = props.intervalType || "Month";
+    let intervalStart: Date;
+    
+    if (dateRangeContext) {
+        intervalType = dateRangeContext.intervalType === "Day" ? "Week" : dateRangeContext.intervalType;
+        intervalStart = dateRangeContext.intervalStart;
+    }
+    else {
+        intervalStart = getDefaultIntervalStart(intervalType, props.weekStartsOn);
```

Comment:
True - current default behaviors are to go back the distance specified.  I didn't catch this discrepancy.  I've updated this to be similar to how the "Month" behavior works.  This will default to the 1st of the month from *5* months ago, making the current month the 6th month displayed in the data.  This is similar to how the Month graph works; it goes to the beginning of *this* month by default, rather than showing you "30 days" of data.

This also exposed an error in how I was generating the ticks for the 6 month graph, I've updated all that code so it's consistent and predictable when the 6 month span *doesn't* start on the 1st of a month.